pkg/api/latest/latest.go

```diff
@@ -117,9 +117,10 @@ func init() {
 	// the list of kinds that are scoped at the root of the api hierarchy
 	// if a kind is not enumerated here, it is assumed to have a namespace scope
 	kindToRootScope := map[string]bool{
-		"Node":      true,
-		"Minion":    true,
-		"Namespace": true,
+		"Node":             true,
+		"Minion":           true,
+		"Namespace":        true,
+		"PersistentVolume": true,
```

Comment:
@quinton-hoole I think the current implementations of VolumeSource supports what you are asking.

Example, Tim was thinking the new NFSMount plugin can stay in VolumeSource as well as be represented in the new PersistentVolumeSource.  He saw re-use for that in a pod beyond an admin-provisioned volume.   In this case, the NFS share is a pet and continues to live outside the lifecycle of a pod.  It can reused by subsequent pods.

Does the correctly address what you are saying?

Also, I don't see a problem with pinning some PVs to a specific namespace (the admin provisions it purposefully that way), except for how the code handles namespaced PVs versus non-namespaced ones.